www/src/pages/index.js

```diff
@@ -319,6 +319,7 @@ export const pageQuery = graphql`
       filter: {
         fields: { starterShowcase: { slug: { in: $featuredStarters } } }
       }
+      sort: { fields: [fields___starterShowcase___slug] }
```

Comment:
@greglobinski this will sort starters for "ecosystem" section on homepage - can this cause unwanted change?

Reply:
Just merged ;)